lib/const.rb

```diff
@@ -122,8 +122,8 @@
 
 CREW_COMMON_FLAGS = '-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold'
 CREW_COMMON_FNO_LTO_FLAGS = '-O2 -pipe -fno-lto -fPIC -fuse-ld=gold'
-CREW_FNO_LTO_LDFLAGS = '-fno-lto'
 CREW_LDFLAGS = '-flto'
+CREW_FNO_LTO_LDFLAGS = '-fno-lto'
```

Comment:
This is the most minor nitpick, but why move this?

Reply:
It makes more sense. before it was
yes
no
no
yes

Now it's
yes
no
yes
no

I don't care, change it back if you like. it just looked better to me.